deps/spidershim/spidermonkey/mozglue/build/moz.build

```diff
@@ -6,10 +6,7 @@
 
 # Build mozglue as a shared lib on Windows, OSX and Android.
 # If this is ever changed, update MOZ_SHARED_MOZGLUE in browser/installer/Makefile.in
-if CONFIG['OS_TARGET'] in ('WINNT', 'Darwin', 'Android'):
-    SharedLibrary('mozglue')
-else:
-    Library('mozglue')
+SharedLibrary('mozglue')
```

Comment:
Hmm, why do you need to do this?  I wonder if there's a better way to fix the issue you're facing?